### DISCUSSION TITLE
Putting rules in other rules

### DISCUSSION THREAD

**Discussion by Tabbourg** [TodePond/CellPond#348] [Help]
In the [Nothing video](https://youtu.be/sQYUQNozljo?si=n33V5G3HxKCkz-UB&t=478) he drags the tadpole into another rule "Just like any other square". How do I do this? Just dragging doesn't seem to work.

**Comment by TodePond** [comment]
when you click on a paddle, the multi-square appears in your toolbar at the top of the screen. or right click and drag the paddle

warning very buggy if you try to make a square that isn't a square
